### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by varopn**

### CODE REVIEW

File: src/checkPassword.test.js [hunk 1]
```diff
@@ -8,12 +8,42 @@ describe(`Function 'checkPassword':`, () => {
   });
 
   it(`should return boolean`, () => {
+    expect(typeof checkPassword('Password')).toBe('boolean');
+  });
+  // using boundary values and equalant classes
```
Comment (iBrianWarner) [inline src/checkPassword.test.js:13]:
```suggestion
```

File: src/checkPassword.test.js [hunk 2]
```diff
@@ -8,12 +8,42 @@ describe(`Function 'checkPassword':`, () => {
...
+    expect(checkPassword('P123456789qwert@')).toBe(true);
+  });
 
+  it(`should return 'false' for the valid password with 17 characters`, () => {
+    expect(checkPassword('Passwordddddddd1!')).toBe(false);
   });
 
-  // write more tests here
+  it(`should return 'false' for the password
+    without digit(s) with 10 characters`, () => {
```
Comment (iBrianWarner) [inline src/checkPassword.test.js:36]:
```suggestion
  it(`should return 'false' for the password without digits`, () => {
```

**Review by iBrianWarner — APPROVED** [review]

File: src/checkPassword.test.js [hunk 3]
```diff
@@ -8,12 +8,42 @@ describe(`Function 'checkPassword':`, () => {
   });
 
   it(`should return boolean`, () => {
+    expect(typeof checkPassword('Password')).toBe('boolean');
+  });
+  // using boundary values and equalant classes
 
+  it(`should return 'true' for the valid password with 9 characters`, () => {
+    expect(checkPassword('Passwor1!')).toBe(true);
```
Comment (iBrianWarner) [inline src/checkPassword.test.js:16]:
```suggestion
    expect(checkPassword('Passwor1!')).toBeTruthy();
```
